### DISCUSSION TITLE
one hobby account multiple domains/projects

### DISCUSSION THREAD

**Discussion by funk10101** [vercel/community#3195] [Feature Question] [answered]
### Question

Is it possible to have 1 hobby account and multiple domain/projects? In other words, can I sign up with one email address for the account and develop my 2 personal sites each with their own domain name under the same account/email? Or does 1 account/email only allow for one domain/project at a time?

**Comment by smaeda-ks** [comment, marked as answer]
you can have multiple projects and domains under your Hobby account
